### DISCUSSION TITLE
23.12.07 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#84] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 나와 서로 친구인 회원 조회 기능 테스트 작성
- 내일은 어떤 일을 하나요?
    - 성능 테스트 방법에 대해 알아보기

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - PR 리뷰 내용 수정 & 공부
  - 플레이리스트에 음악 추가, 제거 단위 테스트 작성
- 내일은 어떤 일을 하나요?
  - 발표 준비
  - 플레이리스트에 음악 추가, 제거 통합 테스트 작성
